docs/modules/operation/pages/quick-start/notification-config.adoc

```diff
@@ -0,0 +1,62 @@
+
+= Step 8: Configure Notifications
+
+This section describes how to configure and customize your {page-component-title} notifications.
+
+NOTE: This step is separate from xref:operation:quick-start/notifications.adoc[setting up default notifications].
+We assume that you have followed the previous steps and are ready to proceed to more in-depth work with notifications.
+
+== Configuration and customization
+
+Now that you've xref:operation:quick-start/baseline.adoc[established your environment's baseline], it's time to take your notes and configure your custom notifications.
+Establishing notifications that are tailored for your {page-component-title} environment will help maintain it.
```

Comment:
I think it's more that the notifications will help them maintain their network, rather than Horizon.
```suggestion
Establishing notifications that are tailored for your environment and monitoring requirements lets the right people see and respond to the right issues in a timely manner.
```